docs/app/_components/SiteMenu.tsx

```diff
@@ -101,16 +302,72 @@ export const SiteMenu = () => {
                 {items.map(page => (
                   <Command.Item
                     className={classNames.item}
-                    key={page.slug}
-                    value={page.slug}
-                    onSelect={() => goto(page.slug)}
+                    key={page.href}
+                    value={page.href}
+                    onSelect={() => window.open(page.href, '_blank')}
+                    keywords={page.keywords}
                   >
-                    {page.title}
+                    <Inline space={4} alignY="center">
+                      {page.name}
+                      <Split />
+                      <span className="text-text-primary-muted text-xs">
+                        <ExternalLink />
+                      </span>
+                    </Inline>
                   </Command.Item>
                 ))}
               </CommandGroup>
             ))}
+            {/* tokens copy command */}
+            {query && (
+              <CommandGroup
+                heading="Colors"
+                key="color"
+                className={classNames.section}
+              >
+                {tokens.map(([token]) => (
+                  <CopyItem
+                    className={classNames.item}
+                    key={token}
+                    value={token}
+                    keywords={['copy']}
+                    copyValue={token.replace('-DEFAULT', '')}
+                  >
+                    {token.replace('-DEFAULT', '')}
+                  </CopyItem>
+                ))}
+              </CommandGroup>
+            )}
+            {/* copy icon command */}
+            {query && (
+              <CommandGroup
+                heading="Icons"
+                key="icons"
+                className={classNames.section}
+              >
+                {Object.values(iconElements).map(elements => (
+                  <CopyItem
+                    key={elements.icon}
+                    value={elements.icon}
+                    keywords={['copy']}
+                    className={classNames.item}
+                    copyValue={elements.svg}
+                  >
+                    {elements.iconElement}
+                    {elements.icon}
+                  </CopyItem>
+                ))}
+              </CommandGroup>
+            )}
           </Command.List>
+          <div className="flex h-10 items-center justify-end gap-4 border-t px-2">
+            <div>
```

Comment:
needs a `flex items-center` so these to are vertically aligned.

Reply:
eh.... what? :D Its already there?

Reply:
in line 363 yes, but not in 364 :P

Reply:
but why? the two divs are just as container for the text inside of it, it is in the middle